### PR TITLE
Change items limit reached message

### PR DESCRIPTION
**Pull request by asdolo**
Follow-up from https://github.com/mojitoinc/mojito-mixers/pull/245#discussion_r833365504

### Preview

image

image

### CODE REVIEW

File: app/src/lib/components/public/CheckoutOverlay/CheckoutOverlay.tsx [hunk 1]
```diff
@@ -322,6 +320,38 @@ export const PUICheckoutOverlay: React.FC<PUICheckoutOverlayProps> = ({
     setWalletAddress(wallet || destinationAddress);
   }, [wallets, destinationAddress, setWalletAddress]);
 
+  // TODO: Extend this to support multiple items
+  const itemCount = useMemo(
```
Comment (Danziger) [inline app/src/lib/components/public/CheckoutOverlay/CheckoutOverlay.tsx:324]: You don't need this as you're using `itemsCount: firstInvoiceItem?.units || 0` as units below, so it probably makes more sense to use that value.

File: app/src/lib/forms/PaymentMethodForm.tsx [hunk 2]
```diff
@@ -362,6 +366,43 @@ export const PaymentMethodForm: React.FC<PaymentMethodFormProps> = ({
   const submitForm = handleSubmit(onSubmit);
   const checkoutErrorMessage = useFormCheckoutError({ formKey: "payment", checkoutError, fields: FIELD_NAMES, setError, deps: [selectedPaymentMethod] });
 
+  const acceptsManyPaymentMethods = acceptedPaymentTypes.length > 1;
+
+  const itemLimitForSelectedPaymentMethod = !!itemLimits && itemLimits[selectedPaymentMethod];
```
Comment (Danziger) [inline app/src/lib/forms/PaymentMethodForm.tsx:371]: Good, but something similar needs to be added in the payment method selector component, as the payment method type might change when selecting a different option from the list.

Comment (asdolo) [reply app/src/lib/forms/PaymentMethodForm.tsx:371]: Solved [here](https://github.com/mojitoinc/mojito-mixers/pull/228/commits/9b10abcde90f368d34cb977f78a4418b432b3921).

**Comment by asdolo** [comment]
@Danziger I moved the logic for the limits message to `useLimits` hook so it can be reused in both `PaymentMethodForm` and `SavedPaymentDetailsSelector`.

**Review by Danziger — APPROVED** [review]

**Pull request merged**